.golangci.yml

```diff
@@ -78,14 +81,23 @@ issues:
         - errcheck
         - dupl
         - gosec
-        - goconst
+    # Allow log.Fatal in main.go (log.go library continues after calling this, unlike std library log.Fatal)
```

Comment:
I don't think this one applies to libraries, as they don't have a `main.go` file, so you can remove this exclusion.